plugins/view-transitions/includes/theme.php

```diff
@@ -365,3 +374,25 @@ function plvt_load_view_transitions(): void {
 	wp_add_inline_script( 'plvt-view-transitions', $init_script );
 	wp_enqueue_script( 'plvt-view-transitions' );
 }
+
+/**
+ * Replaces the animation duration placeholder in the provided CSS with a value based on the transition duration.
+ *
+ * @since n.e.x.t
```

Comment:
Let's mark this one as private since it's very much an internal helper function.
```suggestion
 * @since n.e.x.t
 * @access private
```